### DISCUSSION TITLE
Can i use same extensions / keybindings.json / settings.json?

### DISCUSSION THREAD

**Discussion by renato-ll** [VSCodium/vscodium#1006] [Ideas]
I want to make vscodium read(and write)

~\.vscode\extensions

instead of ~\.vscode-oss\extensions

is it possíble?

Less important but good
keybindings.json / settings.json

In case its possible, it would have any bad side-effects?

**Comment by setanarut** [comment]
you can copy-paste keybindings.json and settings.json or replace files.
